<ui version="4.0" >
 <class>ClientAttributeView</class>
 <widget class="QWidget" name="ClientAttributeView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="leftMargin" >
      <number>2</number>
     </property>
     <property name="topMargin" >
      <number>2</number>
     </property>
     <property name="rightMargin" >
      <number>2</number>
     </property>
     <property name="bottomMargin" >
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="attributeTypeLabel" >
       <property name="text" >
        <string>Vertex Atrribute:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="attributeNameLabel" >
       <property name="text" >
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView" >
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
